ports/psoc6/modules/machine/machine_i2c.c

```diff
@@ -189,3 +218,11 @@ MP_DEFINE_CONST_OBJ_TYPE(
     protocol, &machine_i2c_p,
     locals_dict, &mp_machine_i2c_locals_dict
     );
+
+void mod_i2c_deinit() {
```

Comment:
Also handles for softI2C?

Reply:
GPIO initialization for the softI2C is handled inside mphalport.c using cyhal_gpio_configure( ) function

Reply:
Mmm okay, I think then we have to extend that to initialize that pin if not inited first.